README.md

```diff
@@ -142,6 +142,19 @@ preview.show(fileId, accessToken, {
 | fileVersionId | File version ID to preview. This must be a valid non-current file version ID. Use [Get Versions](https://developer.box.com/reference#view-versions-of-a-file) to fetch a list of file versions |
 | startAt | Object with unit and value properties indicating where to start the preview at. Current supported units are 'seconds' for media and 'pages' for documents. |
 
+**Example**
+
+This configuration sets version `54321` as the preview version for a file with the ID `12345`:
+```
+preview.show(FILE_ID, TOKEN, {
+    fileOptions: {
+        12345: {
```

Comment:
Do you want to use `[FILE_ID]` here or `12345` above? If the file id passed to `.show` is different than this file id, it won't show the specified version.